src/repl.rs

```diff
@@ -44,6 +44,7 @@ pub struct ReplState<F: LurkField> {
 pub struct Repl<F: LurkField, T: ReplTrait<F>> {
     state: T,
     rl: Editor<InputValidator, DefaultHistory>,
+    #[cfg(feature = "with-file-history")]
```

Comment:
This will compile only if the `with-file-history` feature *of the lurk crate* is turned on, and at the moment it isn’t because it does not exist on this crate.

Further, it may be simpler to set your condition on the platform, to check if we managed to customize the rustyline dependency properly.